src/node/routes/pathProxy.ts

```diff
@@ -8,12 +8,12 @@ import { Router as WsRouter } from "../wsRouter"
 
 export const router = Router()
 
-const getProxyTarget = (req: Request, rewrite: boolean): string => {
-  if (rewrite) {
-    const query = qs.stringify(req.query)
-    return `http://0.0.0.0:${req.params.port}/${req.params[0] || ""}${query ? `?${query}` : ""}`
+const getProxyTarget = (req: Request, passthroughPath: boolean): string => {
+  if (passthroughPath) {
+    return `http://0.0.0.0:${req.params.port}/${req.originalUrl}`
   }
-  return `http://0.0.0.0:${req.params.port}/${req.originalUrl}`
+  const query = qs.stringify(req.query)
+  return `http://0.0.0.0:${req.params.port}/${req.params[0] || ""}${query ? `?${query}` : ""}`
```

Comment:
Should we add a test for this?

Reply:
I.e. for the function, like a unit test

Reply:
Will do!

Reply:
nvm, I think in this case it's super simple, there's really no room for error. the existing cli tests test more involved logic and complex flags.

Reply:
Whoops, wrong comment, meant to comment to below.

Reply:
Sounds good!